CMakeLists.txt

```diff
@@ -86,7 +86,7 @@ else()
 endif()
 
 # Set flag for git version to be used as c++ preprocessor macro
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DGIT_HASH=\"\\\"`git describe --always --long --tags --dirty`\\\"\"")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DGIT_HASH=\"\\\"`git -C ${PROJECT_SOURCE_DIR} describe --always --long --tags --dirty`\\\"\"")
```

Comment:
add to `GNUmakefile_full` too

Reply:
GMake builds software "in-place", so it would simply be "`-C ./`" (and the "`PROJECT_SOURCE_DIR`" is specific to CMake).

Reply:
ok, then i think it's fine as is